switchsdk/src/main/java/net/gini/switchsdk/utils/ZoomImageView.java

```diff
@@ -178,13 +178,14 @@ private void init(Context context) {
     }
 
     private enum Mode {
-        NONE, DRAG
+        NONE, DRAG, ZOOM
     }
 
     private class ScaleListener extends ScaleGestureDetector.SimpleOnScaleGestureListener {
 
         @Override
         public boolean onScale(ScaleGestureDetector detector) {
+            mMode = Mode.ZOOM;
```

Comment:
So where do you use it? You only set it.

Reply:
it's not explicitly used, but if it would not be set the mode would be 'DRAG' and in the action move. https://github.com/gini/gini-switch-sdk-android/blob/b6a21a3bdc1a2acd7abd0e3b76211cc229d2445c/switchsdk/src/main/java/net/gini/switchsdk/utils/ZoomImageView.java#L84